.github/workflows/linux-nightly-trunk.yml

```diff
@@ -78,26 +77,21 @@ jobs:
 
     - name: Build package
       run: |
-        echo PATH=$PATH
-        mkdir build && cd build
-        cmake .. -DISPC_PREPARE_PACKAGE=ON -DISPC_INCLUDE_BENCHMARKS=ON
-        make -j4 package
+        .github/workflows/scripts/build-ispc.sh
 
     - name: Sanity testing (make check-all, make test)
       run: |
-        cd build
-        bin/check_isa
-        bin/ispc --support-matrix
-        make check-all
-        make ispc_benchmarks && make test
+        .github/workflows/scripts/check-ispc.sh
 
     - name: Upload package
       uses: actions/upload-artifact@v2
       with:
-        name: ispc_llvm_trunk_linux
+        name: ispc_trunk_linux
```

Comment:
It was intentionally `ispc_llvm_trunk_linux`, as `ispc_trunk_linux` refers to ISPC trunk package (regardless LLVM version used).

Both names are terrible and misleading, but `ispc_llvm_trunk_linux` is a better one.